src/main/groovy/org/jfrog/util/docker/utils/RESTClientNew.groovy

```diff
@@ -0,0 +1,50 @@
+package org.jfrog.util.docker.utils
+
+import groovyx.net.http.HttpResponseDecorator;
+import groovyx.net.http.RESTClient;
+import groovyx.net.http.ResponseParseException;
+import org.codehaus.groovy.runtime.IOGroovyMethods;
+
+/**
+ * Groovy method 'leftShift(OutputStream self, InputStream in)' has deprecated and removed in Groovy 3.
+ * Due to this change, needed to override 'defaultSuccessHandler'.
+ * Source: https://stackoverflow.com/questions/67794538/nosuchmethoderror-from-http-builder-for-binary-content
+ */
+
+public class RESTClientNew extends RESTClient {
```

Comment:
Maybe you should consider another name then "New"? maybe V2? in a few months from now it will not be "New". Also, what would be the next client name? "New New"?

Reply:
@dorsJfrog